lib/active_model/serializer/adapter/attributes.rb

```diff
@@ -2,6 +2,11 @@ module ActiveModel
   class Serializer
     module Adapter
       class Attributes < Base
+        def initialize(serializer, options = {})
+          super
+          @include_tree = IncludeTree.from_include_args(options[:include] || '*')
```

Comment:
gotchya.  What if the class-level method to get the include associations tree per-adapter? adapters could just override that method if they need to, and we could have a generic one in base?